packages/boot/test/upgrading/test-upgrade-vats.js

```diff
@@ -74,7 +74,7 @@ test('upgrade vat-board', async t => {
   const { bfile } = t.context;
   const bundles = {
     board: {
-      sourceSpec: bfile('../../src/vat-board.js'),
+      sourceSpec: bfile('../../../vats/src/vat-board.js'),
```

Comment:
reaching across packages seems awkward.

`bfile` doesn't grok package specifiers? hm. Is it worth using `importSpec`?